src/book/03-guides/02-tools/03-webots.mdx

```diff
@@ -363,3 +363,45 @@ From the root of the NUWebots repository run
 ```
 
 Where `<name of controller>` is the name you want to use for the new controller. A folder and files will be created with this name, so they must be valid file names (don't use spaces in your controller name).
+
+## Recording a Game
+
+NUbots runs full Webots games to benchmark the current performance of the code. The following steps describe how to create a video of a full game.
+
+1. Ensure you have followed the previous steps [Install Webots](#install-webots) and [Running the Official RoboCup Simulation in Webots](#running-the-official-robocup-simulation-in-webots).
+
+2. Set up the game configuration.
+
+   1. Open the [game.json file](https://github.com/NUbots/NUWebots/blob/main/webots/projects/samples/contests/robocup/controllers/referee/game.json).
+   2. Change the `"blue":` `"id"` to `12`.
+   3. Change the `"blue":` `config:` field to `"team_1.json"`.
+
+   At the moment, there is an issue due to the fact that both teams have the same team ID since they are both NUbots teams. A penalty to one robot will also penalise the corresponding robot on the other team.
```

Comment:
This deserves an issue on NUbots IMO. We should be trying to fix this

Reply:
Do we just need to change the team_id [here](https://github.com/NUbots/NUbots/blob/main/module/support/configuration/GlobalConfig/data/config/GlobalConfig.yaml#L2) to sort this?

Reply:
Yep, that just needs to be different for the robots on different teams.